templates/firefox-ios.xliff

```diff
@@ -2059,6 +2063,42 @@
         <source>Okay</source>
         <note>Restore Tabs Affirmative Action</note>
       </trans-unit>
+      <trans-unit id="Onboarding.IntroAction.v102" xml:space="preserve">
+        <source>Get Started</source>
+        <note>Describes the action on the first onboarding page in our Onboarding screen. This string will be on a button so user can continue the onboarding.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.IntroDescription.v102" xml:space="preserve">
+        <source>Indie. Non-profit. For good. Committed to the promise of a better Internet for everyone.</source>
+        <note>Description for the first onboarding page in our Onboarding screens.</note>
```

Comment:
Can you explain in the comments what Indie means? I have no idea - so I imagine some other people too

Reply:
I added some explanation, let me know if it's better!